deepreg/predict.py

```diff
@@ -148,21 +148,21 @@ def predict_on_dataset(
 
 
 def build_config(
-    config_path: Union[str, List[str]], log_root: str, log_dir: str, ckpt_path: str
+    config_path: Union[str, List[str]], log_dir: str, exp_name: str, ckpt_path: str
 ) -> Tuple[Dict, str, str]:
     """
     Function to create new directory to log directory to store results.
 
-    :param config_path: string or list of strings, path of configuration files
-    :param log_root: str, root of logs
-    :param log_dir: string, path to store logs.
-    :param ckpt_path: str, path where model is stored.
+    :param config_path: path of configuration files.
+    :param log_dir: path of the directory.
+    :param exp_name: name of the experiment.
```

Comment:
use searchable "log directory" and "experiment name"

Reply:
updated